packages/picasso.js/src/core/chart-components/ref-line/lines-and-labels.js

```diff
@@ -61,46 +122,55 @@ export function alignmentToNumber(align) {
  * @param {object[]} items - Array of all items (for collision detection)
  * @ignore
  */
-export function createLineWithLabel({ chart, blueprint, renderer, p, settings, items }) {
+export function createLineWithLabel({ chart, blueprint, renderer, p, settings, items, slopeLine }) {
   let doesNotCollide = true;
   let line = false;
+  let slope = false;
   let rect = false;
   let label = false;
   let value = false;
   let style = extend(true, {}, settings.style.line, p.line || {});
+  let slopeStyle = extend(true, refLabelDefaultSettings(), settings.style.label || {}, { fill: style.stroke });
```

Comment:
What is the difference in styling for a sloped line? Could they not share the same settings?

Reply:
As the label was not created in picasso before instead in scatter plot, so we had no label styling but now as we render it here we have styles sent for label from scatter plot.

Reply:
I dont fully understand, could you share an example definition that sets the slope styling and the other styling?

Reply:
When looking at this line `et item = extend(true, refLabelDefaultSettings(), settings.style.label || {}, { fill: style.stroke }, p.label);` there seem to be a label styling for non-sloped line label already.

I guess the question is then, is there a difference in the API how these are set? Or are they set the same but resolved differently?

Reply:
p.label may be used in other charts but not in scatter plot and now with sloped lines the way they are resolved is different compared to the existing p.label..